cg/meta/workflow/mip_dna.py

```diff
@@ -67,3 +70,17 @@ def config_sample(
     def get_gene_panel(self, case_id: str) -> list[str]:
         """Create and return the aggregated gene panel file."""
         return self._get_gene_panel(case_id=case_id, genome_build=GENOME_BUILD_37)
+
+    def get_managed_variants(self) -> list[str]:
+        """Create and return the managed variants."""
+        return self._get_managed_variants(genome_build=GENOME_BUILD_37)
+
+    def write_managed_variants(self, case_id: str, content: list[str]) -> None:
+        """Write the managed variants to case dir."""
+        out_dir = Path(self.root, case_id)
+        out_dir.mkdir(parents=True, exist_ok=True)
+        WriteFile.write_file_from_content(
+            content="\n".join(content),
+            file_format=FileFormat.TXT,
+            file_path=Path(out_dir, f"managed_variants{FileExtensions.VCF}"),
```

Comment:
```suggestion
            file_path=Path(out_dir, f"managed_variants{FileExtensions.VCF}"),
```
`managed_variants` repeated string

Reply:
Replaced by StrEnum